tests/components/buienradar/test_sensor.py

```diff
@@ -0,0 +1,43 @@
+"""The tests for the Dark Sky platform."""
+import unittest
+
+from homeassistant.setup import setup_component
+from homeassistant.components import sensor
+
+from tests.common import get_test_home_assistant
+
+CONDITIONS = ["stationname", "temperature"]
+BASE_CONFIG = {
+    "sensor": [
+        {
+            "platform": "buienradar",
+            "name": "volkel",
+            "latitude": 51.65,
+            "longitude": 5.7,
+            "monitored_conditions": CONDITIONS,
+        }
+    ]
+}
+
+
+class TestBuienradarSensorSetup(unittest.TestCase):
```

Comment:
New test must be standalone pytest test functions.

See eg:
https://github.com/home-assistant/home-assistant/blob/5a592f1291d37dab5b5cd514e2841d7343085700/tests/components/manual/test_alarm_control_panel.py#L83-L119

Reply:
Ack. I prefer this style (I bootstrapped this from another component). Have applied this.